### PR TITLE
Airtable services

### PR DESCRIPTION
**Pull request by kosiakkatrina**
**What**  
Update airtable gateway to use new table structure
Update airtable resources mock
Concatenate all resources (fss and Airtable) before displaying them
Add council tag to the Airtable resources

Before:
image

After:
image

**Why**  
To make more resources available through better conversations

**Anything else?**  
This is assuming that all the Airtable resources will be assigned to FSS categories, otherwise they will not show up on the page

**Trello card**
https://trello.com/c/gL9FxGdn

Co-authored-by: @elena-vi

### CODE REVIEW

File: components/Feature/VulnerabilitiesGrid/ResourceCard/index.js [hunk 1]
```diff
@@ -32,9 +32,9 @@ const ResourceCard = ({
   const trimLength = (s, length) => s.length > length ? s.substring(0, length) + "..." : s
 
   const selfReferralElement = (selfReferral == 'No') ? 'Referral required' : 'Self referral'
-  const websiteElement = websites && websites.length > 0 &&  websites.map(website => (<a href={websites[0]} target="_blank" rel="noopener noreferrer">{websites[0]}</a>))
+  const websiteElement = websites && websites.length > 0 &&  websites.map(website => (<><a href={website} target="_blank" rel="noopener noreferrer">{website}</a><br/></>))
```
Comment (elena-vi) [inline components/Feature/VulnerabilitiesGrid/ResourceCard/index.js:35]: that tag didn't look quite right, I think it should be something like this
```suggestion
  const websiteElement = websites && websites.length > 0 &&  websites.map(website => (<a href={website} target="_blank" rel="noopener noreferrer">{website}</a><br/></>))
```

File: components/Feature/VulnerabilitiesGrid/ResourceCard/index.js [hunk 2]
```diff
@@ -32,9 +32,9 @@ const ResourceCard = ({
...
   const distributionElement =  tags.filter(t => HIDDEN_TAGS.includes(t)).join(", ")
-  const tagsElement = tags.filter(t => !HIDDEN_TAGS.includes(t)).map(item=> (<span key={"tags-"+item} className={`${css.tags} tag-element`}>{trimLength(item, 20)}</span>))
+  const tagsElement = tags.filter(t => !HIDDEN_TAGS.includes(t)).map(item=> (<span key={"tags-"+item} className={`${css.tags} tag-element ${css[`${item}-tag`]}`}>{trimLength(item, 20)}</span>))
```
Comment (elena-vi) [inline components/Feature/VulnerabilitiesGrid/ResourceCard/index.js:37]: I'm a little bit confused by the `css.tags` bit, are you accessing actual css here or is css just some property name?

Comment (elena-vi) [reply components/Feature/VulnerabilitiesGrid/ResourceCard/index.js:37]: also it looks like it was there already so we might want to rename that as part of separate pr

**Review by elena-vi — APPROVED** [review]
looks good apart from that lil tag 😄

File: components/Feature/VulnerabilitiesGrid/ResourceCard/index.js (hunk 2 above)
Comment (kosiakkatrina) [reply components/Feature/VulnerabilitiesGrid/ResourceCard/index.js:37]: The css is something that we're importing at the top `import css from './index.module.scss';`
And then we're accessing different scss parts from there.
I think this is the way Next makes you do styling

File: components/Feature/VulnerabilitiesGrid/ResourceCard/index.js (hunk 1 above)
Comment (kosiakkatrina) [reply components/Feature/VulnerabilitiesGrid/ResourceCard/index.js]: I've just put it inside a div and removed fragment and `</br>` 😅

File: components/Feature/VulnerabilitiesGrid/ResourceCard/index.js (hunk 2 above)
Comment (kosiakkatrina) [reply components/Feature/VulnerabilitiesGrid/ResourceCard/index.js:37]: https://nextjs.org/docs/basic-features/built-in-css-support#adding-component-level-css that's how nextjs handles component level css

**Review by kosiakkatrina — COMMENTED** [review]
Remember to add env vars

**Pull request merged**